subsys/net/lib/download_client/Kconfig

```diff
@@ -26,6 +26,14 @@ config NRF_DOWNLOAD_CLIENT_STACK_SIZE
 	int "Thread stack size"
 	default 2048
 
+config NRF_DOWNLOAD_CLIENT_SOCK_TIMEOUT_MS
+	int "Receive timeout, in millisec"
```

Comment:
can you write the whole word here, i.e. "milliseconds"

Reply:
Done